GuildedRoseLLCUITests/GuildedRoseLLCUITests.swift

```diff
@@ -10,7 +10,7 @@ class GuildedRoseLLCSpec: QuickSpec {
             app = XCUIApplication()
             app.launch()
         }
-        describe("the message") {
+        describe("the greeting message") {
             context("for the greeting") {
                 it("is 'Welcome to the Guilded Rose'"){
```

Comment:
I would avoid duplicating the text of the message in the test description. I would fear the test falling out of date.